projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -59,7 +59,12 @@ <h3 *ngIf="content.title" class="cx-card-title">
         </button>
       </div>
       <div class="col-md-6">
-        <button class="btn btn-block btn-primary" (click)="delete()">
+        <button
+          #deleteBtn
```

Comment:
We introduced the `cxFocus` directive back in version `2.0` which had an auto-focus feature for when elements are added to the DOM. Could you investigate to see if we could use this here instead? It could remove the need for adding focus logic to the component itself.

Autofocus feature: https://sap.github.io/spartacus-docs/auto-focus/